packages/utils/telemetry-utils/src/errorLogging.ts

```diff
@@ -378,6 +387,20 @@ export class LoggingError extends Error implements ILoggingError, Omit<IFluidErr
         }
     }
 
+    /**
+    * Determines if a given object is an instance of a LoggingError
+    * @param object - any object
+    * @returns - true if the object is an instance of a LoggingError, false if not.
+    */
+    public static typeCheck(object: unknown): object is LoggingError {
```

Comment:
Let's get some quick unit tests covering this

Reply:
Whoops, forgot to add this back in from the old PR. Sounds good. I have added the changes in commit #a1d49538.